### ISSUE TITLE
TestUtils.displayFile(String Filename) fails on linux

### ISSUE THREAD

**Issue by brianberzins** [approvals/ApprovalTests.Java#251]
https://github.com/approvals/ApprovalTests.Java/blob/c1021bb96e644f6045d2ce6966cd131af0a955e9/approvaltests-util/src/main/java/com/spun/util/tests/TestUtils.java#L146-L167

"open" is macOS specific and won't work out of the box on ubuntu installation.

Honestly don't know how deep you feel this is going, but I'm trying to think of reasonable additions to support Ubuntu. Using file separators won't be sufficient to distinguish between macOS and a linux distro, but I also understand that trying to support a bunch of linux distros might not be easy in this case. `xdg-open` seems like the most appropriate command if you want to support most linux distros.

**Comment by LarsEckart** [comment]
I sent you an invite for tomorrow, maybe you can join us? We kinda lack a linux machine to develop on. If not, let me know, maybe I can find someone else.

**Comment by brianberzins** [comment]
I should be able to make it

**Comment by brianberzins** [comment]
Confirmed fixed in 15.1.0

Also showed the customized openers work worked with this:
``` Java 
import com.spun.util.tests.TestUtils;

TestUtils.registerOpener(f -> LinuxOpener.executeOnLinux(f, "firefox"));
```
